src/relay/transforms/pattern_util.h

```diff
@@ -270,7 +270,7 @@ inline Constant MakeConstantScalar(DataType dtype, T value) {
  */
 template <typename T>
 static inline Constant MakeConstantTensor(DataType dtype, std::vector<int64_t> shape,
-                                          std::vector<T> value) {
+                                          T value) {
```

Comment:
I would suggest you create another template for `Array`.

Reply:
Could you update this?

Reply:
Sorry for late. Updated